http/resource.go

```diff
@@ -19,6 +19,24 @@ import (
 	"github.com/filebrowser/filebrowser/v2/fileutils"
 )
 
+var resourceGetSizeHandler = withUser(func(w http.ResponseWriter, r *http.Request, d *data) (int, error) {
+	folder, err := files.NewFileInfo(files.FileOptions{
+		Fs:         d.user.Fs,
+		Path:       r.URL.Path,
+		Modify:     d.user.Perm.Modify,
+		Expand:     true,
+		FolderSize: true,
```

Comment:
This feature should be configurable and disabled by default. It might be harmful for public instances.